src/ScottPlot5/ScottPlot5/Plottables/Signal.cs

```diff
@@ -19,13 +19,22 @@ public class Signal : IPlottable
 
     public Color Color = new(0, 0, 255);
     public float LineWidth = 1;
+    public string? Label { get; set; }
```

Comment:
General API question, should we favor nullable strings or `string.Empty` for plottables without labels?

Are there strong pros/cons for either strategy?

Reply:
There isn't a great way to note that a string can or cannot be empty through the type system, unless of course you create your own type for it.

Now arguably there isn't a much better way to note that a string can or cannot be null through the type system, given that non-nullable strings can still be null.

I personally prefer null, because trying to work with a null string is going to fail much faster and more obviously than trying to work with an empty string. You might consider an empty string invalid, but most operations on empty strings are going to work fine and make bugs much trickier to track down.

Reply:
> There isn't a great way to note that a string can or cannot be empty through the type system

> I personally prefer null, because trying to work with a null string is going to fail much faster and more obviously than trying to work with an empty string

I'm sold! I'll lean toward nullable strings to signal they're optional

> You might consider an empty string invalid

I like this suggestion too... I'll think about it, but it's on my radar now.

Thanks for your input!